### PR TITLE
man: update man page

### PR DESCRIPTION
**Pull request by apahim**
man is quite outdated. This patch updates the man to reflect the
current options and UI.

Reference: https://trello.com/c/UB6hG2mD
Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

**Comment by apahim** [comment]
[BUG: man page should contain all command line options](https://trello.com/c/UB6hG2mD/901-bug-man-page-should-contain-all-command-line-options)

**Comment by apahim** [comment]
[BUG: man page should contain all command line options](https://trello.com/c/UB6hG2mD/901-bug-man-page-should-contain-all-command-line-options)

File: man/avocado.rst [hunk 1]
```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
...
+                         --mux-out instead)
 
 
 RUNNING A TEST
 ==============
 
 The most common use of the `avocado` command line tool is to run a test::
 
- $ avocado run sleeptest
+ $ avocado run sleeptest.py
```
Comment (ldoktor) [inline man/avocado.rst]: Nice touch

File: man/avocado.rst [hunk 2]
```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
...
 
-This command will run the `sleeptest` test, as found on the standard test
+This command will run the `sleeptest.py` test, as found on the standard test
 directories. The output should be similar to::
 
- JOB ID    : <id>
- JOB LOG   : /home/<user>/avocado/job-results/job-<date>-<shortid>/job.log
```
Comment (ldoktor) [inline man/avocado.rst]: Wasn't this more accurate? (no strong opinion)

File: man/avocado.rst [hunk 3]
```diff
@@ -6,8 +6,8 @@
 SYNOPSIS
 ========
 
- avocado [-h] [-v] [--config CONFIG_FILE]
- {run,list,sysinfo,multiplex,plugins,datadir} ...
+ avocado [-h] [-v] [--config [CONFIG_FILE]] [--show [STREAM[:LVL]]] [-s]
```
Comment (ldoktor) [inline man/avocado.rst]: there is (and was) extra space at the beginning which increases the offset of the first paragraph.

File: man/avocado.rst [hunk 4]
```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
 options. Most other options are implemented via plugins and will depend
 on them being loaded::
```
Comment (ldoktor) [inline man/avocado.rst]: Well the output looks like this:

```
SYNOPSIS
       avocado  [-h] [-v] [--config [CONFIG_FILE]] [--show [STREAM[:LVL]]] [-s] {config,diff,distro,exec-path,list,multiplex,plugins,run,sysinfo}
       ...

DESCRIPTION
       Avocado is a modern test framework that is built on the experience accumulated with autotest (http://autotest.github.io).

       avocado is also the name of its test runner command line tool, described in this man page.

       For more information about the Avocado project, please check its website: http://avocado-framework.github.io/

OPTIONS
       The following list of options are builtin, application level avocado options. Most other options are  implemented  via  plugins  and  will
       depend on them being loaded:

          -h, --help            show this help message and exit
          -v, --version         show program's version number and exit
          --config [CONFIG_FILE]
```

I think it'd be useful to repeate the `avocado -h` command at the beginning of the OPTIONS as the DESCRIPTION makes it a bit distant. Also below you use `$ avocado multiplex --help` followed by the output, which is easier to follow.

File: man/avocado.rst [hunk 5]
```diff
@@ -28,62 +28,106 @@ The following list of options are builtin, application level `avocado`
...
 
  $ avocado multiplex --help
- usage: avocado multiplex [-h] [--filter-only [FILTER_ONLY [FILTER_ONLY ...]]]
-                          [--filter-out [FILTER_OUT [FILTER_OUT ...]]] [-t]
-                          [-c]
-                          multiplex_files [multiplex_files ...]
-
- positional arguments:
-   multiplex_files       Path(s) to a multiplex file(s)
+ usage: avocado multiplex [-h] [--system-wide] [-c] [-d] [-t] [-i]
```
Comment (ldoktor) [inline man/avocado.rst]: Actually I believe there should be another category for multiplex. Currently it's part of `OPTIONS` as the only command. The rest of the commands have their own categories...

Comment (ldoktor) [inline man/avocado.rst]: And it should most probably be moved below, `RUNNING A TEST` looks like more important section to me...

File: man/avocado.rst [hunk 6]
```diff
@@ -122,8 +158,10 @@ Edit your `~/.config/avocado/avocado.conf` file and add::
 Running the same example with this option will give you::
 
     $ scripts/avocado run sleeptest --show-job-log
-    START sleeptest.py
-    PASS sleeptest.py
+    ...
+    START 1-sleeptest.py:SleepTest.test
+    PASS 1-sleeptest.py:SleepTest.test
```
Comment (ldoktor) [inline man/avocado.rst]: On line 235 | 273:

```
  $ ls -1 ~/avocado/job-results/latest/test-results/
 sleeptest.py
```
should be `1-sleeptest.py` (or so)

Comment (ldoktor) [inline man/avocado.rst]: and in the next paragraph as well...

File: man/avocado.rst [hunk 7]
```diff
@@ -610,17 +639,16 @@ Here is how to run the sleeptest example test in a remote machine with IP
...
- JOB LOG   : /home/<user>/avocado/job-results/job-<date>-<shortid>/job.log
- TESTS     : 1
- (1/1) sleeptest.py:  PASS (1.01 s)
- RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
- TESTS TIME : 1.01 s
+    JOB ID     : 4733dc1638b78c9af90187ca91f547058edf8c14
+    JOB LOG    : /home/<user>/avocado/job-results/job-2017-03-03T10.33-4733dc1/job.log
+    LOGIN      : fedora@localhost:22 (TIMEOUT: 60 seconds)
+     (1/1) sleeptest.py:SleepTest.test: PASS (1.02 s)
+    RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
```
Comment (ldoktor) [inline man/avocado.rst]: on line 667 | 695

```
 BUGS
 ====
 
If you find a bug, please report it over our github page as an issue.
```

I'd suggest adding a link to our github.

Comment (ldoktor) [inline man/avocado.rst]: Also above we have section `FILES`, how about list `/etc/avocado/conf.d` as well?

**Pull request closed without merging**